src/main/java/ai/elimu/rest/v2/crowdsource/ContributorsRestControllerWeb3.java

```diff
@@ -58,7 +58,7 @@ public String handleGetRequest(
         
         JSONObject jsonObject = new JSONObject();
         
-        String providerIdWeb3 = contributorJSONObject.getString("providerIdWeb3");
+        String providerIdWeb3 = contributorJSONObject.getString("providerIdWeb3").toLowerCase();
```

Comment:
Let's move this to _after_ the `isBlank()` check, to prevent _NullPointerException_?

Reply:
you are right